Jellyfin.Api/Auth/BaseAuthorizationHandler.cs

```diff
@@ -38,35 +39,35 @@ public abstract class BaseAuthorizationHandler<T> : AuthorizationHandler<T>
         }
 
         /// <summary>
-        /// Validate authenticated claims.
+        /// Validate basic authentication claims.
         /// </summary>
         /// <param name="claimsPrincipal">Request claims.</param>
+        /// <param name="user">The found user or null.</param>
         /// <param name="ignoreSchedule">Whether to ignore parental control.</param>
-        /// <param name="localAccessOnly">Whether access is to be allowed locally only.</param>
-        /// <param name="requiredDownloadPermission">Whether validation requires download permission.</param>
-        /// <returns>Validated claim status.</returns>
+        /// <returns>Validated claim status and the found user.</returns>
         protected bool ValidateClaims(
             ClaimsPrincipal claimsPrincipal,
-            bool ignoreSchedule = false,
-            bool localAccessOnly = false,
-            bool requiredDownloadPermission = false)
+            out User? user,
+            bool ignoreSchedule = false)
         {
+            user = null;
+
             // ApiKey is currently global admin, always allow.
             var isApiKey = claimsPrincipal.GetIsApiKey();
             if (isApiKey)
             {
                 return true;
             }
 
-            // Ensure claim has userId.
+            // Get user id from claim.
             var userId = claimsPrincipal.GetUserId();
-            if (userId.Equals(default))
+            if (userId.Equals(Guid.Empty))
```

Comment:
I'm pretty sure @Bond-009 converted to `.Equals(default)` because it has better performance

Reply:
Correct